plugins/chat/assets/javascripts/discourse/routes/chat-threads.js

```diff
@@ -0,0 +1,10 @@
+import { inject as service } from "@ember/service";
```

Comment:
When we click on a my threads in the sidebar we want to be sure the current channel is reset